src/components/Book.jsx

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+export default class Book extends Component {
+  render() {
+    return (
+      <div>
+        <h1>
+          Books
+          {this.props.name} 
+        </h1>
+        <button type="submit">Remove</button>
+      </div>
+    );
+  }
```

Comment:
- [ ] For this component, a book needs to have a title and author props. You're only having the author prop. Kindly make work on that. 👍